domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapperTest.java

```diff
@@ -1869,6 +1878,46 @@ void testMapEmptySearchPartyEntity() {
 
     }
 
+    @Nested
+    @DisplayName("Should return Category model whose fields match those in the CategoryEntity")
+    class MapCategoryTests {
+
+
+        @Test
+        void testMapCategoryEntity() {
+            CategoryEntity categoryEntity = new CategoryEntity();
+            categoryEntity.setCaseType(caseTypeLiteEntity("CaseTypeReference"));
+
+
+            Category category = classUnderTest.map(categoryEntity);
+
+            assertEquals(category.getCategoryId(), categoryEntity.getCaseType().getReference());
+            assertEquals(category.getCategoryLabel(), categoryEntity.getCategoryLabel());
+            assertEquals(category.getParentCategoryId(), categoryEntity.getParentCategoryId());
+            assertEquals(category.getLiveFrom(), categoryEntity.getLiveFrom());
+            assertEquals(category.getLiveTo(), categoryEntity.getLiveTo());
+            assertEquals(category.getDisplayOrder(), categoryEntity.getDisplayOrder());
+            assertEquals(category.getCaseTypeId(), categoryEntity.getCaseType());
```

Comment:
These assertions are the wrong way round - `assertEquals()` should have the `expected` as the first argument - and a couple aren't checking the correct thing (e.g. first is comparing `categoryId` to the case type's reference, last one is comparing the `caseTypeId` to an object rather than the reference).